src/utils.ts

```diff
@@ -36,6 +37,16 @@ export function buildUrl(
   body.push(`URL:${event.url || document.URL}`);
   body.push(`DTSTART:${formatDate(event.startTime)}`);
   body.push(`SUMMARY:${event.title}`);
+  if (event.attendees) {
+    event.attendees.map(attendee => {
+      const regExp = attendee.match(/(?:"?([^"]*)"?\s)?(?:<?(.+@[^>]+)>?)/);
```

Comment:
For better semantic meaning, it should be like this. 

```js
const regExp = /(?:"?([^"]*)"?\s)?(?:<?(.+@[^>]+)>?)/;
const matches = attendee.match(regExp);
```

Reply:
btw, what's your understanding to the regular expression above? It's actually much more capable than I expected.